Applications/ApplicationsLib/TestDefinition.cpp

```diff
@@ -215,16 +233,26 @@ TestDefinition::TestDefinition(BaseLib::ConfigTree const& config_tree,
         std::string const relative_tolerance_parameter =
             "--rel " + relative_tolerance;
 
-        //
-        // Construct command line.
-        //
-        std::string command_line =
-            vtkdiff + " -a " + safeString(field_name) + " -b " +
-            safeString(field_name) + " " + safeString(reference_filename) +
-            " " + safeString(output_filename) + " " +
-            absolute_tolerance_parameter + " " + relative_tolerance_parameter;
-        INFO("Will run '{:s}'", command_line);
-        _command_lines.emplace_back(std::move(command_line));
+        for (auto &filename : filenames)
+        {
+            std::string const& output_filename =
+                BaseLib::joinPaths(output_directory, filename);
+            _output_files.push_back(output_filename);
+            // TODO (naumov) expand filename relative to ref path for globbing.
+            std::string const& reference_filename =
+                BaseLib::joinPaths(reference_path, filename);
```

Comment:
This looks OK to me. What do you mean by "expand filename relative to reference path"?

Reply:
You added this line: https://github.com/ufz/ogs/blob/master/Applications/ApplicationsLib/TestDefinition.cpp#L184

I guess this line can be removed now?

Reply:
Oh, that was me :) Who knows what it should mean.
Can be dropped.